app/helpers/application_helper.rb

```diff
@@ -23,6 +23,11 @@ def home_page?
     params.values_at(:controller, :action) == %w[pages index]
   end
 
+  def open_petition_page?
+    # XXX What should go here? Same as petition_page? plus petition.open? Is that allowed here?
+    true
```

Comment:
Ah, great, thanks, I still don't get my head around where variables are magically available in ruby :)